ui/admin/src/assets/i18n/en.json

```diff
@@ -118,13 +121,13 @@
   "Street": "Street",
   "City": "City",
   "No result found.": "No result found.",
-  "monday": "monday",
```

Comment:
I'm not sure that the translation mechanism should be used to apply an uppercase. (this has been probably already used at other places). We can use CSS rules for this.

Reply:
Yes, in some place the CSS rule is a good method, but we already encounters issues with facets. It's then difficult to find the right selector to use `::first-letter`.